### ISSUE TITLE
Skip action: deployer-user

### ISSUE THREAD

**Issue by chandresh** [HashNuke/ansible-elixir-stack#15]
I am new to Ansible (as well as Elixir & Phoenix) and am trying to deploy my first test app to production. I tried ansible-elixir-stack and get the following error at the action deployer-user:

```
fatal: [*.*.*.*] => SSH Error: Permission denied (publickey,password).
    while connecting to *.*.*.*
It is sometimes useful to re-run the command using -vvvv, which prints SSH debug output to help diagnose the issue.
```

Is there a way to skip the action: deployer-user? I already have a deployer user setup on my production server so I do not need this.

**Comment by HashNuke** [comment]
@chandresh No it cannot be skipped. You can configure the name of the deployer user so that it's different from your other user. The reason is because ansible-elixir-stack uses the `asdf` version manager to install erlang, elixir & node.js (people might be using other stuff for other apps). Don't want things to overlap.

**Comment by HashNuke** [comment]
@chandresh Here's the configuration details - https://github.com/HashNuke/ansible-elixir-stack/blob/master/docs/configuration.md

**Comment by chandresh** [comment]
@HashNuke Thanks for quick response. I checked the configuration already and there is no option to skip the deployer-user. There is an option to skip the postgres. For now I have forked the project and locally updated the code for action-setup.yml as follows:

```
- when: setup_deployer == True
  include: deployer-user.yml
```

I am still playing with this. If you want I can submit a PR once I have a solution that works.
